com.xiaoyaocz.simple_live.yaml

```diff
@@ -0,0 +1,221 @@
+app-id: com.xiaoyaocz.simple_live
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: simple_live
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    version: "24.08"
+    directory: lib/ffmpeg
+    add-ld-path: .
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /share/man
+  - /share/doc
+cleanup-commands:
+  - mkdir -p ${FLATPAK_DEST}/lib/ffmpeg
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.ScreenSaver
+  # Temporary dummy fixup for nm.dart
+  # - --system-talk-name=this.is.a.dummy.bus.name
+  - --system-talk-name=org.freedesktop.NetworkManager.IP4Config
```

Comment:
what is this for?

Reply:
* `org.freedesktop.ScreenSaver` prevents the screen from dimming or locking during live streaming playback.
* `org.freedesktop.NetworkManager.IP4Config` retrieves the device's current IP address. It's used in the app's feature that synchronizes followed channels and configurations with other Simple Live clients on the local network.